### ISSUE TITLE
Failure when running bundle install as airflow user

### ISSUE THREAD

**Issue by dkinzer** [tulibraries/ansible-role-airflow#5]
From `/var/lib/airflow/tul_cob` 

```
bundle install
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    current directory: /var/lib/airflow/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/unf_ext-0.0.7.5/ext/unf_ext
/var/lib/airflow/.rbenv/versions/2.4.1/bin/ruby -r ./siteconf20190403-30967-waln2k.rb extconf.rb
checking for -lstdc++... no
creating Makefile

current directory: /var/lib/airflow/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/unf_ext-0.0.7.5/ext/unf_ext
make "DESTDIR=" clean

current directory: /var/lib/airflow/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/unf_ext-0.0.7.5/ext/unf_ext
make "DESTDIR="
compiling unf.cc
make: g++: Command not found
make: *** [unf.o] Error 127

make failed, exit code 2

Gem files will remain installed in /var/lib/airflow/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/unf_ext-0.0.7.5 for inspection.
Results logged to /var/lib/airflow/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/extensions/x86_64-linux/2.4.0-static/unf_ext-0.0.7.5/gem_make.out

An error occurred while installing unf_ext (0.0.7.5), and Bundler cannot continue.
Make sure that `gem install unf_ext -v '0.0.7.5' --source 'https://rubygems.org/'` succeeds before bundling.
```

If I try to install that specific gem
```
gem install unf_ext -v '0.0.7.5'
Building native extensions.  This could take a while...
ERROR:  Error installing unf_ext:
	ERROR: Failed to build gem native extension.

    current directory: /var/lib/airflow/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/unf_ext-0.0.7.5/ext/unf_ext
/var/lib/airflow/.rbenv/versions/2.4.1/bin/ruby -r ./siteconf20190403-20269-zbj4kp.rb extconf.rb
checking for -lstdc++... no
creating Makefile

current directory: /var/lib/airflow/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/unf_ext-0.0.7.5/ext/unf_ext
make "DESTDIR=" clean

current directory: /var/lib/airflow/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/unf_ext-0.0.7.5/ext/unf_ext
make "DESTDIR="
compiling unf.cc
make: g++: Command not found
make: *** [unf.o] Error 127

make failed, exit code 2

Gem files will remain installed in /var/lib/airflow/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/unf_ext-0.0.7.5 for inspection.
Results logged to /var/lib/airflow/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/extensions/x86_64-linux/2.4.0-static/unf_ext-0.0.7.5/gem_make.out```

**Comment by dkinzer** [comment]
ok, I see that issue is that `g++` command is not found So I'll try to add that to fix it.

**Comment by dkinzer** [comment]
I'm suppose to add this I think ... but where?

```
- name: install the 'Development tools' package group
  yum:
    name: "@Development tools"
    state: present
```

**Comment by cmharlow** [comment]
Fixed at role level via #6
